source/common/router/config_impl.cc

```diff
@@ -62,7 +62,7 @@ HedgePolicyImpl::HedgePolicyImpl(const envoy::api::v2::route::HedgePolicy& hedge
       hedge_on_per_try_timeout_(hedge_policy.hedge_on_per_try_timeout()) {}
 
 HedgePolicyImpl::HedgePolicyImpl()
-    : initial_requests_(1), additional_request_chance_({}), hedge_on_per_try_timeout_(false) {}
+    : initial_requests_(1), additional_request_chance_(), hedge_on_per_try_timeout_(false) {}
```

Comment:
Can you just remove this initializer entirely and rely on the default?

Reply:
Thanks!